builds/misc/templates/build-api-proxy.yaml

```diff
@@ -0,0 +1,23 @@
+parameters:
+  name: 'API proxy'
+  imageName: 'api-proxy-module'
```

Comment:
I am confused, as we discussed, you agreed to change these parameters to use variables or even not defined here as it is default value in script/linux/buildAPIProxy.sh. Why still here?

Reply:
You can't create variable in template: https://github.com/MicrosoftDocs/azure-devops-docs/issues/6890